react/w-books/src/app/screens/Login/styles.module.scss

```diff
@@ -50,6 +52,11 @@
   width: 90%;
 }
 
+.login-loading-button {
+  background-color: lighten($color: $green, $amount: 10);
+  cursor: disabled;
+}
```

Comment:
if you are validating the disabled attribute in your HTML tag, maybe you can do it like: 
```scss
.button {
  ...styles
  &:disabled { 
    background-color: lighten($color: $green, $amount: 10);
    cursor: disabled;
  }
}
```